deepchem/models/torch_models/progressive_multitask.py

```diff
@@ -6,14 +6,15 @@
 from deepchem.utils.pytorch_utils import get_activation
 
 from collections.abc import Sequence as SequenceCollection
-from typing import List, Tuple, Callable
+from typing import List, Tuple, Callable, Literal, Union
 
 
 class ProgressiveMultitask(nn.Module):
```

Comment:
@gauthamk02 Shouldn't this abstract `TorchModel` class or do we have a separate `ProgressiveMultitaskModel` class?

Reply:
I have added a separate `ProgressiveMultitaskModel` class inheriting `TorchModel` in PR #3822 . I am waiting for this to be merged so I can rebase the changes in this PR to it